### PR TITLE
feat - home page, navbar and "page not found" page

### PR DESCRIPTION
**Pull request by prachi-sahani**
This PR includes the home page for the notes app and the "page not found" page. Both the pages are responsive

[Home page preview](https://cakeopedia-bjso0jn41-prachi-sahani.vercel.app/)

[Page not found preview](https://cakeopedia-bjso0jn41-prachi-sahani.vercel.app/notes)

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      </Link>
+      {!(
+        location.pathname.includes("login") ||
+        location.pathname.includes("signup")
+      ) && (
+        <div className="search">
+          <button className="btn-icon material-icons ">search</button>
+          <input type="text" className="search-input" />
+          <button className="btn-icon material-icons ">close</button>
+        </div>
```
Comment (Hamdrive) [inline src/components/Navbar.jsx:19]: Interesting implementation for dynamically show/hide searchbar

File: src/stylesheets/home..css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.container {
```
Comment (Hamdrive) [inline src/stylesheets/home..css]: file name has a typo, should be `home.css` instead of `home..css` (two dots)

**Review by Hamdrive — COMMENTED** [review]
Nice clean code written, UI looks good
Suggested fix for CSS filename, also should look into use of CSS modules, 
else current use of unique CSS classnames also works

**Review by aryan-13 — APPROVED** [review]
Good folder structure, Clean code. Looks Good to merge

File: src/stylesheets/home..css (hunk 2 above)
Comment (prachi-sahani) [reply src/stylesheets/home..css]: Done!

**Pull request merged**